setup.cfg

```diff
@@ -45,3 +45,6 @@ skip=
     envs
     migrations
     common/lib/safe_lxml/safe_lxml/etree.py
+    wsgi.py
+    wsgi_apache_lms.py
+known_first_party=calc,capa,chem,dogstats_wrapper,safe_xml,symmath,xmodule
```

Comment:
Although these packages are our code, they are in edx-platform as their own installable things.  Should they be considered like "third-party but friends"?

Reply:
[isort doesn't support a category like that.](https://github.com/timothycrosley/isort/wiki/isort-Settings) Do you think it makes sense that these would be treated like third-party modules instead of first-party?

I saw instances of both in the codebase before I ran isort so I wasn't sure which to pick.